pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -597,6 +640,17 @@ func ignoreValidatingAdmissionPolicyMatchConditions(new, old *admissionregistrat
 	return true
 }
 
+// ignoreMutatingAdmissionPolicyMatchConditions returns true if there have been no updates that could invalidate previously-valid match conditions
+func ignoreMutatingAdmissionPolicyMatchConditions(new, old *admissionregistration.MutatingAdmissionPolicy) bool {
+	if !reflect.DeepEqual(new.Spec.ParamKind, old.Spec.ParamKind) {
+		return false
+	}
+	if !reflect.DeepEqual(new.Spec.MatchConditions, old.Spec.MatchConditions) {
```

Comment:
Fixed. I'll change this for ValidatingAdmissionPolicy too, which looks safe since it is used for ratcheting and semantic equality widens which expressions can be ratcheted.